### PR TITLE
Cert Print Guidelines

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@frrrances

**Comment by mhoeber** [comment]
@srpearce can you check

**Comment by frrrances** [comment]
:+1:

File: en_us/open_edx_students/source/front_matter/change_log.rst [hunk 1]
```diff
@@ -2,6 +2,20 @@
 Change Log
 ############
 
+*****************
+September 2015
```
Comment (srpearce) [inline en_us/open_edx_students/source/front_matter/change_log.rst]: October

File: en_us/open_edx_students/source/front_matter/change_log.rst [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+*****************
+
+.. list-table::
+   :widths: 15 70
+   :header-rows: 1
+
+   * - Date
+     - Change
+   * - 8 Sep 2015
```
Comment (srpearce) [inline en_us/open_edx_students/source/front_matter/change_log.rst]: Sep --> Oct

File: en_us/students/source/front_matter/change_log.rst [hunk 3]
```diff
@@ -12,6 +12,9 @@ Change Log
 
    * - Date
      - Change
+   * - 8 Sep 2015
```
Comment (srpearce) [inline en_us/students/source/front_matter/change_log.rst]: Oct (make new table)

**Comment by srpearce** [comment]
:+1: modulo changing September to October

**Pull request merged**